packages/website/components/Hero.tsx

```diff
@@ -9,8 +9,9 @@ export const Hero = () => {
     classNameLight: 'theme-default',
```

Comment:
I think here if youre just interested in getting the `darkMode.value` you don't need to pass in the config object

I believe it can be just `useDarkMode()`